13-TeV-examples/uproot_python/HZZAnalysis.ipynb

```diff
@@ -208,6 +210,7 @@
     "fraction = 1.0 # reduce this is if you want the code to run quicker\n",
     "                                                                                                                                  \n",
     "#tuple_path = \"Input/4lep/\" # local \n",
+    "#tuple_path = \"https://resources.opendata.atlas.cern/datasets/Legacy13TeV/4lep/\" # web address\n",
```

Comment:
I get a 403 trying to visit this page. Should it work? If not, should we remove it?

Reply:
It should work, as it's the new path for the samples (I see currently there seems to be some problem), but the code won't work with it as there are not yet certificates in place for it. I put it there as a sort of placeholder waiting for a future fix where we remove the comment and the older path. However, if you find it confusing I can remove this newer non-working commented path and keep only the working older one.

Reply:
That's ok — if the plan is that it should work, then no issue keeping it!